doc/manual/configuration.xml

```diff
@@ -1165,7 +1165,7 @@ uams_dhx2.so -&gt; uams_dhx2_pam.so</programlisting> then you're using PAM,
 aclmode = passthrough</screen>
 
           <para>For an explanation of what this knob does and how to apply it,
-          check your hosts ZFS documentation (eg man zfs).</para>
+          check your hosts ZFS documentation (e.g man zfs).</para>
```

Comment:
Nitpick: This should also have the trailing period. "e.g" -> "e.g."

Reply:
Thanks, fixed.